boards/arm/qemu_cortex_a53/qemu_cortex_a53.dts

```diff
@@ -14,7 +14,7 @@
 
 	psci {
 		compatible = "arm,psci-0.2";
-		method = "hvc";
+		method = "smc";
```

Comment:
This will break in the `!SMP` case.

You can easily check that switching to `SMC` and run the PSCI test `tests/arch/arm/arm_psci` in the `!SMP` case. That test is failing for `SMC` but it is successful for `HVC`, and this is because QEMU is actually emulating an EL2 hypervisor.

This brings me again to the question: why are you switching to `virtualization=on` in the SMP case.

Reply:
Not sure, I could not recall. Seems no need virtualization=on here.